tests/phpunit/integration/Modules/Idea_HubTest.php

```diff
@@ -135,14 +145,14 @@ public function test_draft_labels() {
 		$this->idea_hub->set_post_idea( $post2->ID, $idea );
 
 		// With an IdeaHub post
-		$post_states2 = apply_filters( 'display_post_states', array( 'draft' => 'Draft' ), $post2 );
+		$post_states2 = apply_filters( 'display_post_states', array( 'publish' => 'Draft' ), $post2 );
 		// With a regular draft post
 		$post_states3 = apply_filters( 'display_post_states', array( 'draft' => 'Draft' ), $post3 );
 
 		$post_states1 = apply_filters( 'display_post_states', array( 'draft' => 'Draft' ), $post1 );
 
 		$this->assertEquals( $post_states1, array( 'draft' => 'Idea Hub Draft “Using Site Kit to analyze your success”' ) );
-		$this->assertEquals( $post_states2, array( 'draft' => 'Idea Hub Draft “Why Penguins are guanotelic?”' ) );
+		$this->assertEquals( $post_states2, array( 'publish' => 'inspired by Idea Hub' ) );
```

Comment:
Nit: I generally avoid using numbered variables to separate cases in a test unless it's specific to the test in some way such as asserting something is order-specific.

The quick and easy change to make here would be to use more expressive names for the variables themselves (e.g. `$idea_hub_post_published`, `$regular_post_draft`, etc). Ideally, the cases would be extracted into a data provider so that each test run is only for one case at a time (no special names needed). I think this would be the better use here since the cases here are independent of one another.